docs/pages/router/reference/link-preview.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: Link Preview
+description: Learn how to add a preview to your link on iOS.
+---
+
+import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
+
+<ContentSpotlight
+  alt="A video presenting the link preview feature in Expo Router."
+  file="/expo-router/link-preview.mp4"
```

Comment:
```suggestion
  file="expo-router/link-preview.mp4"
```

The video is not showing on the page. Let's fix the path above.

<img width="1139" height="698" alt="CleanShot 2025-08-11 at 17 43 58" src="https://github.com/user-attachments/assets/502ec133-42f5-4a6d-947d-517bb0f1d191" />